mautrix_telegram/commands/auth.py

```diff
@@ -227,6 +227,42 @@
         return await evt.reply("Unhandled exception while sending password. "
                                "Check console for more details.")
 
+@command_handler(needs_auth=True,
+                management_only=True,
+                help_section=SECTION_AUTH,
+                help_text="Log in with a bot token.")
+async def bot_login(evt: CommandEvent):
+    if await evt.sender.is_logged_in():
+        return await evt.reply("You are already logged in.")
+    if evt.config["appservice.public.enabled"] && evt.config.get("bridge.allow_matrix_login", True):
```

Comment:
I'm not sure what the settings exactly mean and if I'm doing the right checks

Reply:
`appservice.public.enabled` checks if the public website (only used for login) is enabled.

Also, `&&` isn't a thing in Python.